python/pyspark/pandas/tests/test_dataframe.py

```diff
@@ -108,13 +108,117 @@ def test_dataframe(self):
             self.assert_eq(pdf, psdf)
 
     def test_creation_index(self):
```

Comment:
Can we also test with various data types other than integer type ?? (string and timestamp ?)

Reply:
Done

Reply:
Nice 👍